packages/material-ui/src/ButtonBase/ButtonBase.test.js

```diff
@@ -521,6 +521,36 @@ describe('<ButtonBase />', () => {
         fireEvent.click(getByTestId('trigger'));
         expect(container.querySelectorAll('.ripple-pulsate')).to.have.lengthOf(1);
       });
+
+      it('should stop the ripple on blur even if disableTouchRipple is set', () => {
```

Comment:
The "even" is a bit unclear. `disableTouchRipple` does not affect the focus ripple so the behavior is expected i.e. the prop combination is valid.

```suggestion
      it('should stop the ripple on blur if disableTouchRipple is set', () => {
```